sale_order_team_from_product/readme/CONTRIBUTORS.md

```
@@ -0,0 +1,3 @@
+* [Cetmix](https://cetmix.com/):
```

markdownlint-cli2 (0.18.1)

First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)